packages/node_modules/@webex/webex-core/test/integration/spec/services/services.js

```diff
@@ -460,13 +464,17 @@ describe('webex-core', () => {
         catalog._unloadServiceUrls('preauth', [testUrl]);
       });
 
-      it('marks a host as failed and removes from host list', () => {
-        const priorityUrl = services.get(testUrlTemplate.name, true);
-        const originalServiceUrlHostLength = testUrl.hosts.length;
+      it('marks a host as failed', () => {
+        const priorityServiceUrl = catalog._getUrl(testUrlTemplate.name);
+        const priorityUrl = priorityServiceUrl._getPriorityHostUrl();
 
         services.markFailedUrl(priorityUrl);
 
-        assert.isBelow(testUrl.hosts.length, originalServiceUrlHostLength);
+        const failedHost = priorityServiceUrl.hosts.find(
+          (host) => host.failed
+        );
+
+        assert.isDefined(failedHost);
```

Comment:
Could we be a bit more specific in this assertion and check that `failedHost.url === priorityUrl`?

Reply:
I believe I have an implementation for this, generating it now.